webapp/src/serial.tsx

```diff
@@ -147,6 +149,18 @@ export class Editor extends srceditor.Editor {
         }
     }
 
+    dropStaleCharts() {
+        let now = Util.now()
+        for (let i = 0; i < this.charts.length; ++i) {
```

Comment:
use ``this.charts.forEach(chart => ...)``

Reply:
Fixed